src/sparseml/transformers/sparsification/obcq/sparse_gpt_modifier.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from typing import Optional
+
+import torch
+
+from sparseml.pytorch.sparsification.modifier import BaseModifier, PyTorchModifierYAML
+from sparseml.transformers.sparsification.obcq.layer_compressor import LayerCompressor
+
+
+__all__ = ["SparseGPTModifier"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@PyTorchModifierYAML()
+class SparseGPTModifier(BaseModifier):
+    def __init__(
+        self,
+        sparsity: float = 0.5,
+        block_size: int = 128,
+        quantize: bool = True,
+        dampening_frac: Optional[float] = 0.01,
+        sequential_update: Optional[bool] = True,
```

Comment:
for serialization, need to mark these as `@ModifierProp`s (see other modifiers for example). also good to add a serialization test (see other modifiers for example)